website/docs/r/firewall.html.markdown

```diff
@@ -64,7 +64,9 @@ The following arguments are supported:
 
 * `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
 
-* `ip_configuration` - (Required) A `ip_configuration` block as documented below.
+* `ip_configuration` - (Required) An `ip_configuration` block as documented below.
+
+* `management_ip_configuration` - (Optional) A `management_ip_configuration` block as documented below, which allows force-tunnelling of traffic to be performed by the firewall. Changing this forces a new resource to be created.
```

Comment:
I'm still not sure about the wording. You can change the `management_ip_configuration` block without recreating the firewall. It only forces a new resource when you change the `subnet_id`. Adding or removing the block does force a new resource ofcourse

Reply:
@dragetta I think you pretty much got it there! How about:
```suggestion
* `management_ip_configuration` - (Optional) A `management_ip_configuration` block as documented below, which allows force-tunnelling of traffic to be performed by the firewall. You can add and remove this block, however changing `subnet_id` in an existing block forces a new resource to be created.
```

Reply:
@manicminer I've adjusted your wording a bit. Let me know if this is ok.

Reply:
@draggeta briiliant 👍